extensions/tracing/src/main/java/io/opentelemetry/extension/tracing/ExtendedTracer.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.extension.tracing;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Scope;
+import java.util.concurrent.Callable;
+
+/** Provides easy mechanisms for wrapping standard Java constructs with an OpenTelemetry Span. */
+@SuppressWarnings("InconsistentOverloads")
+public class ExtendedTracer {
```

Comment:
I think it would be nice to make this "implement" the Tracer interface as well for users to not have to store 2 of them.

Reply:
This seems pretty nice, can delegate the methods

Reply:
will do! good idea!